sdk/src/main/kotlin/io/rover/sdk/ui/blocks/image/ViewImage.kt

```diff
@@ -25,7 +26,19 @@ internal class ViewImage(
         if (binding != null) {
             val measuredSize = binding.measuredSize ?: throw RuntimeException("ViewImage may only be used with a view model binding including a measured size (ie. used within a Rover screen layout).")
 
-            imageView.alpha = 0f
+            with(imageView) {
+
+                alpha = 0f
+                val isAccessible = binding.viewModel.isClickable || !binding.viewModel.isDecorative
+                isFocusableInTouchMode = isAccessible
+                importantForAccessibility = if (isAccessible) {
+                    imageView.contentDescription = binding.viewModel.accessibilityLabel
```

Comment:
[Super Nitpick] Could be `contentDescription = binding.viewModel.accessibilityLabel` like you're doing with alpha but ignore this if you're doing it to be more explicit about what you're setting contentDesc on